feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.ts

```diff
@@ -92,19 +95,38 @@ export class CustomerTicketingCreateDialogComponent
   }
 
   createTicketRequest(): void {
+    this.attachment = this.form.get('file')?.value;
     if (this.form.invalid) {
       this.form.markAllAsTouched();
       FormUtils.deepUpdateValueAndValidity(this.form);
     } else {
       this.subscription = this.customerTicketingFacade
         .createTicket(this.getCreateTicketPayload(this.form))
-        .subscribe({
-          complete: () => {
+        .subscribe((response: any) => {
```

Comment:
1) Instead of using Subscribe inside a subscribe,  please use:
  .createTicket(payload)
  .pipe(
    tap(response) => {
      .uploadAttachmennt(params)
    }  
  )
  .subscribe({ complete: fn(), error: fn() })


refer to "reopenRequest" function at:  https://github.com/SAP/spartacus/blob/a6e29ea516c3b88f2ba56c70c3828e9fe6e9855c/feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen-dialog/customer-ticketing-reopen-dialog.component.ts

2. Please create an interface to host createTicket response.  Using any is a bad practice

Reply:
Fixed https://github.com/SAP/spartacus/pull/16383/commits/c062aee83c661029a3cd1f580623c2740696708c, https://github.com/SAP/spartacus/pull/16383/commits/ad76584c21a1d588ef40fcb528e5207d31abddbe